### ISSUE TITLE
Load command causes read amplification

### ISSUE THREAD

**Issue by wwq2333** [Alluxio/alluxio#17599]
**Alluxio Version:**
2.9.x

**Describe the bug**
use `fs load` to warm up the dataset, we found that `fs load` displayed bandwidth is significantly lower than the k8s monitoring bandwidth.

![image](https://github.com/Alluxio/alluxio/assets/12540485/289025d0-575b-4b42-89fe-b576d10b73c6)

4 worker pod , network receive: `sum(rate(container_network_receive_bytes_total{container_label_io_kubernetes_pod_namespace="xx", container_label_io_kubernetes_pod_name=~"alluxio-worker-.*"}[1m])) by (container_label_io_kubernetes_pod_name, instance)`
![image](https://github.com/Alluxio/alluxio/assets/12540485/f1245e5c-59fd-40b8-b618-a3db3d3aa988)

**Expected behavior**
the network consumption monitored by k8s is slightly larger than the loaded data.

**Additional context**
- this is a large file dataset (1100+ files, 650GB) case, and there is no such problem with warming up image small file datasets.
- alluxio.worker.block.store.type= FILE

**update**
I captured packets in one of the workers and found that the cut-off range of the range request is abnormal .
alluxio.user.block.size.bytes.default=4194304.
without considering the boundary case, the range request should be `Range: bytes=4MB*n - 4MB*(n+1)-1`, this is exactly one block size.
in fact, one of the range request/response is as follows:
```
GET /prefix/target HTTP/1.1
Range: bytes=113246208-9223372036854775806
.....

HTTP/1.1 206 Partial Content
Content-Length: 1151639521
Content-Range: bytes 113246208-1264885728/1264885729
....
```
It can be clearly found that the size of the response is much larger than the size of a block;

**Comment by jiacheliu3** [comment]
Hi @wwq2333 Could you clarify what your k8s monitor graph shows? Is that network upload+download for each worker pod? Or just upload?

**Comment by Haoning-Sun** [comment]
@jja725 @jiacheliu3 2.x Is it feasible to refer to the implementation of pr #16637  in version 2.x ?

**Comment by Haoning-Sun** [comment]
> @jja725 @jiacheliu3 2.x Is it feasible to refer to the implementation of pr #16637 in 2.x ?

@LuQQiu Can you provide some advice?

**Comment by jja725** [comment]
@Haoning-Sun Yes, we would welcome some contributions from the community to help us cherry-pick some of the code changes from #16637 and apply them to UfsIOManager(which would be used by Load). Last time I discussed with @wwq2333 that he would like to contribute.
